AutomatedTesting/Gem/PythonTests/Performance/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+if(NOT PAL_TRAIT_BUILD_TESTS_SUPPORTED OR NOT PAL_TRAIT_BUILD_HOST_TOOLS)
+    return()
+endif()
+
+ly_add_pytest(
+    NAME AutomatedTesting::EditorLevelLoadingPerfTests_DX12
+    TEST_SUITE main
```

Comment:
I don't think it is appropriate to add 10+ minutes of perf testing to the main suite. Suggest it should be added to either the benchmark or periodic suite.

Testing it out in Jenkins using this configuration makes sense, but leaving this enabled creates a significant increase in locally executing the main suite.

Reply:
Which machines are configured for which pipelines would be a question for @sig-build, to my knowledge the GPU machines can be made available elsewhere but require extra configuration